CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ A more detailed list of changes is available in the corresponding milestones for
 ## 0.8.0 (2020-Jul-??)
 ### Note-worthy code changes
   - remove PriorityLevel class as it makes classifying checks by priority more complicated then necessary! (issue #2981)
-
+  - Added **[com.google.fonts/check/varfont_duplicate_instance_names]**: Avoid duplicate instance names in variable fonts
```

Comment:
please add a reference to issue the corresponding issue being fixed in your changelog entry just like we usually do. In this case it is issue #2988

Reply:
Wait, this seems wrong. This PR is #2988, but there's an actual issue: https://github.com/googlefonts/fontbakery/issues/2986
Which one did you mean?

Reply:
oops! Good catch!
I used github's autocompletion and ended up pointing here LOL

The issue I really wanted to reference was #2986